src/components/Header/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from 'styled-components';
+import Colors from '../../variables';
```

Comment:
```suggestion
import React from 'react';
import styled from 'styled-components';
import Colors from 'variables';
```

Can be changed to this :)